pipelines/resource_pipeline.go

```diff
@@ -230,6 +230,11 @@ func (Pipeline) CustomizeSchema(s *common.CustomizableSchema) *common.Customizab
 	s.SchemaPath("edition").SetCustomSuppressDiff(common.EqualFoldDiffSuppress)
 	s.SchemaPath("storage").SetCustomSuppressDiff(suppressStorageDiff)
 
+	// As of 6th Nov 2024, the DLT API only normalizes the catalog name when creating
+	// a pipeline. So we only ignore the equal fold diff for the catalog name and not other
+	// UC resources like target or ingestion_definition.connection_name.
+	s.SchemaPath("catalog").SetCustomSuppressDiff(common.EqualFoldDiffSuppress)
```

Comment:
Can you check if it's also done for the `schema` field?

Reply:
Thanks, done and updated the comment. `schema` is not normalized.

Reply:
it's really interesting that schema isn't normalized...